src/Raven.Server/Documents/Expiration/ExpirationStorage.cs

```diff
@@ -124,21 +133,21 @@ private void ThrowWrongExpirationDateFormat(Slice lowerId, string expirationDate
                                 if (cancellationToken.IsCancellationRequested)
                                     return expired;
 
-                                var clonedId = multiIt.CurrentKey.Clone(context.Transaction.InnerTransaction.Allocator);
+                                var clonedId = multiIt.CurrentKey.Clone(options.Context.Transaction.InnerTransaction.Allocator);
 
                                 try
                                 {
-                                    using (var document = _database.DocumentsStorage.Get(context, clonedId, DocumentFields.Id | DocumentFields.Data))
+                                    using (var document = _database.DocumentsStorage.Get(options.Context, clonedId, DocumentFields.Id | DocumentFields.Data | DocumentFields.ChangeVector))
```

Comment:
Does the `DocumentFields.ChangeVector` needed here?

Reply:
Not right now, however, the next issue is to check if the change vector for this document has been replicated to the other side. 
We decided to skip it for this issue, but we'll get back to it.